doc/nmstate.service.8.in

```diff
@@ -9,8 +9,21 @@ nmstate\&.service
 nmstate\&.service invokes \fBnmstatectl service\fR command which
 apply all network state files ending with \fB.yml\fR in
 \fB/etc/nmstate\fR folder.
-The applied network state file sha256 digest will be stored at \fB.applied\fR
-file to prevent repeated applied on next service start.
+
+By default, The applied network state file will be renamed with postfix
+\fB.applied\fR to prevent repeated applied on next service start.
+
+With \fB/etc/nmstate/nmstate.conf\fR holding below content:
+
+\fB
+[service]
+delete_applied = false
+\fR
+
+The nmstate.service will not rename applied network state, but store sha256
+digest at \fB.applied\fR file to prevent repeated applied on next service
+start.
+
```

Comment:
```suggestion
By default, the network state file will be renamed with the suffix
\fB.applied\fR after applying it. This is to prevent it being applied again when the service runs next.

To prevent deleting the original file, add the following to the 
\fB/etc/nmstate/nmstate.conf\fR file:

\fB
[service]
delete_applied = false
\fR
```